### DISCUSSION TITLE
Inline CSS Variable with URL

### DISCUSSION THREAD

**Discussion by ghost** [parcel-bundler/parcel#5338] [Q&A]
How can I set the same behavior of `--bg: url(img/tour.jpg);` → `--bg: url(tour.04dd8a7b.jpg);` with inline CSS?

For instance:
```html
<div style="--bg: url(img/tour.jpg);"></div>
```
compile to:
```html
<div style="--bg: url(tour.04dd8a7b.jpg);"></div>
```

**Comment by mischnic** [comment]
If I recall correctly, this should already be the case...

**Comment by ghost** [comment]
Sorry, I forgot to mention Parcel's version: `^2.0.0-nightly.443` (I ran `npm update`).

I couldn't find anything in the docs using keywords like: inline css, css variables, html css, etc. Is this and expected behavior for this version?

Here is an example:
![image](https://user-images.githubusercontent.com/23178554/98878372-32ecc880-2459-11eb-983b-545a9e758286.png)

Inline variable stays the same with or without the variable declared n the CSS file, which in this case is under the class `.tournaments__tournament`.

----

I guess this will come to a conclusion once I move to a templating framework (like `lit-html`), with the help of `import img from "img/tour.jpg";`.

My use case is/was that I just wanted a quick preview ready for my client, without any framework on top, now I want to know if it's possible.
